src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -178,7 +180,7 @@ public async Task GetAsync_CancelDuringResponseBodyReceived_Buffered_TaskCancele
                         clientFinished.SetResult(true);
                         await serverTask;
                     } catch { }
-                });
+                }).WithCancellation(timeoutSource.Token);
```

Comment:
`LoopbackServerFactory.CreateServerAsync` already has a timeout parameter, and it's shorter than 2min:

```c#
public abstract Task CreateServerAsync(Func<GenericLoopbackServer, Uri, Task> funcAsync, int millisecondsTimeout = 60_000, GenericLoopbackOptions options = null);
```

Reply:
Unfortunately, it's not used.
```C#
public override Task CreateServerAsync(Func<GenericLoopbackServer, Uri, Task> funcAsync, int millisecondsTimeout = 60_000, GenericLoopbackOptions options = null)
{
    return LoopbackServer.CreateServerAsync((server, uri) => funcAsync(server, uri), options: CreateOptions(options));
}
```
https://github.com/dotnet/runtime/blob/6f6c94dbb2259deb083eefb755698cc8e0f0a4ce/src/libraries/Common/tests/System/Net/Http/LoopbackServer.cs#L885

Reply:
oof! it might be better to fix the parameter than to tack on a new layer?

Reply:
Yes, it was might first thought, but I'm afraid of breaking other tests because this parameter has a default value so the fix will affect all callers. However, if you think this should be fixed, I will do it tomorrow.

Reply:
The timeout is there to prevent runaway tests; I think it would be better to enable it and then investigate individual tests that exceed it. Our tests pass pretty quickly so hopefully those will be minimal.

Reply:
The CI timeout is pretty long 10-15 minutes. We can also adjust the default on slow platforms like ARM (if we need to)